app/components/modal-catalog-launch/template.hbs

```diff
@@ -0,0 +1,79 @@
+  <div class="row form-group">
+    <div class="col-sm-12 col-md-3 form-label">
+      <label class="form-control-static">Template</label>
+    </div>
+    <div class="col-sm-12 col-md-7 r-mb15">
+      {{new-select
+        classNames='form-control'
+        content=versionsArray
+        prompt="Choose a Template"
+        optionLabelPath="version"
+        optionValuePath="link"
+        value=selectedTemplate}}
+    </div>
+    {{#if loading}}
+      <div class="col-md-12 text-center r-mt15">
+        <i class="fa fa-spinner fa-spin" style="font-size:36px;"></i>
+      </div>
+    {{/if}}
+    {{#if selectedTemplateModel }}
+      <div class="col-sm-12 col-md-3 form-label">
+        <label class="form-control-static">Name*</label>
+      </div>
+      <div class="col-sm-12 col-md-7 r-mb15">
+        {{input type='text' class='form-control r-mb10' value=selectedTemplateModel.name}}
+      </div>
+      <div class="col-sm-12 col-md-3 form-label">
+        <label class="form-control-static">Description*</label>
+      </div>
+      <div class="col-sm-12 col-md-7 r-mb15">
+        {{input type='text' class='form-control r-mb10' value=selectedTemplateModel.description}}
+      </div>
+      <div class="col-md-12 col-sm-12 over-hr r-mt20 r-mb10">
+        <span>
+          Template Questions
+        </span>
+      </div>
+      <form {{action 'submitTemplate' on='submit'}}>
+        <div class="row form-group">
+          {{#each selectedTemplateModel.questions as |question|}}
+            <div class="col-sm-12 col-md-7 col-md-offset-3">
+              <label class="form-label form-control-static">{{question.description}}</label>
+              {{#if (or (eq question.type 'string') (eq question.type 'int'))}}
+                {{input type='text' class='form-control r-mb10' value=question.answer}}
+              {{else}}
+                {{#if (eq question.type 'boolean')}}
+                  <div class="r-mb10">
+                    <label class="radio-inline">
+                      {{radio-button selection=question.answer value=true}} True
+                    </label>
+                    <label class="radio-inline">
+                      {{radio-button selection=question.answer value=false}} False
+                    </label>
+                  </div>
+                {{else}}
+                  {{#if (eq question.type 'enum')}}
+                    <select name="" class="form-control r-mb10" onChange={{action (mut question.answer) value="target.value"}}>
+                      <option value="">Choose an option...</option>
+                      {{#each question.options key="@identity" as |option|}}
+                        <option value={{option}}>{{option}}</option>
+                      {{/each}}
+                    </select>
+                  {{/if}}
+                {{/if}}
+              {{/if}}
+            </div>
+          {{/each}}
+        </div>
+        <div class="text-center">
+          {{partial "add-cancel"}}
```

Comment:
Why not `save-cancel` ("Create")?  Actually, I'm not sure why `add-cancel` exists still, there seems to be no other use of it.